src/util/source_location.cpp

```diff
@@ -93,3 +94,25 @@ std::ostream &operator << (
   out << source_location.as_string();
   return out;
 }
+
+/*******************************************************************\
+
+Function: is_build_in
```

Comment:
s/build/built/

Do we really not have this as a utility?  If not then we really really should add it.